Formula/bashate.rb

```diff
@@ -0,0 +1,32 @@
+class Bashate < Formula
+  include Language::Python::Virtualenv
+
+  desc "Code style enforcement for bash programs"
+  homepage "https://github.com/openstack-dev/bashate"
+  url "https://github.com/openstack-dev/bashate/archive/0.5.1.tar.gz"
+  sha256 "44fa6645fd7d5169c40747714986f1093ed24e2474e4a368c2695bf339c1551e"
+
+  depends_on "python@2" if MacOS.version <= :snow_leopard
+
+  resource "pbr" do
+    url "https://files.pythonhosted.org/packages/source/p/pbr/pbr-4.0.4.tar.gz"
+    sha256 "a9c27eb8f0e24e786e544b2dbaedb729c9d8546342b5a6818d8eda098ad4340d"
+  end
+
+  def install
+    venv = virtualenv_create(libexec)
```

Comment:
virtualenv_install_with_resources

Reply:
So, I can't do that. If I do that, then the PBR_VERSION gets set while pbr is being installed, and then pbr's version is set to 0.5.1, which then means the bashate install doesn't work properly. I need it to be set after pbr has been installed, but before bashate, hence the split-out version of `virtualenv_install_with_resources`

Reply:
OK. If that's the case, it's an upstream bug that must be fixed before we can merged this. It doesn't appear to affect a build from HEAD but there has been no release since Apr 26, 2016.